# The name of the ROS node sending this message.
string origin_node_name

# The objects themselves are stored in an array
# (i.e. a vector). The number of elements in the 
# array is obtained using objects.size() in C++.
MovingObject[] objects